packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -24,17 +24,90 @@ <h2>
+                            data-description="{{:: preset.description }}"
+                            data-model="$ctrl.job.image"
+                            data-picture="{{:: preset.logo }}"
+                            data-label="{{:: preset.name }}"
+                            data-variant="light"
+                            data-on-change="$ctrl.generateName()"
+                            data-values="[preset]"
+                            data-required
+                        >
+                            <oui-select-picker-section
+                                class="oui-select-picker__section"
+                            >
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{:: preset.link }}"
+                                >
+                                    <span
+                                        data-translate="pci_projects_project_training_jobs_submit_select_image_preset_info"
+                                    ></span>
+                                    <span
+                                        class="oui-icon oui-icon-external-link"
+                                        aria-hidden="true"
+                                    ></span>
+                                </a>
+                            </oui-select-picker-section>
+                        </oui-select-picker>
+                    </div>
+                    <div class="col-md-6 col-xl-4 my-3">
+                        <oui-select-picker
+                            data-name="missing"
+                            data-description="{{:: 'pci_projects_project_training_jobs_submit_select_image_preset_request' | translate }}"
+                            data-picture=""
+                            data-label="{{:: 'pci_projects_project_training_jobs_submit_select_image_preset_missing_framework' | translate }}"
+                            data-variant="light"
+                            data-values="[preset]"
+                            data-required
+                            data-model="$ctrl.job.image"
+                            data-on-change="$ctrl.generateName()"
+                        >
+                            <oui-select-picker-section
+                                class="oui-select-picker__section"
+                            >
+                                <a
+                                    target="_blank"
+                                    class="oui-button oui-button_ghost"
+                                    rel="noopener"
+                                    data-ng-href="{{ $ctrl.discordUrl }}"
+                                >
+                                    <span>OVHcloud Discord</span>
+                                    <i
```

Comment:
Same comment than previous one:
```suggestion
                                    <span
```

Reply:
done